client/comparison-extension/package.json

```diff
@@ -45,6 +42,12 @@
     "semver": "^6.3.0",
     "typescript": "~4.5.5"
   },
+  "peerDependencies": {
+    "@theia/core": "^1.34.1",
+    "@theia/process": "^1.34.1",
+    "@theia/workspace": "^1.34.1",
+    "@theia/git": "^1.34.1"
+  },
```

Comment:
Are we using any new features that require this version? I am not an expert, but from my understanding, this prevents people from using the application with theia versions lower than the one specified here. So maybe we shouldnt't be too restrictive?

Reply:
Thanks, yes that's totally true, I left them at their previous value now (`^1.0.0)